test/artefacts/maternity-paternity-calculator/adoption/maternity/2018-04-05/2018-04-06/yes/no/yes/2018-04-06/2018-01-01/2017-11-01/every_2_weeks/3000.0/4/usual_paydates/2018-01-01.txt

```diff
@@ -6,8 +6,8 @@ The employee is entitled to Statutory Adoption Leave.
 
 |Leave                         | Key dates                    |
 |------------------------------|------------------------------|
-|Start                         |  6 April 2018          |
-|End                           |  4 April 2019            |
+|Start                         | 6 April 2018          |
```

Comment:
In this file we've got two approached to markdown tables:

formatted (broken)

```
|Leave                         | Key dates                    |
|------------------------------|------------------------------|
|Start                         | 6 April 2018          |
|End                           | 4 April 2019            |
|Latest date to [claim leave](/employers-paternity-pay-leave/)     |   |
```

and inline

```
Date | SAP amount
-|-
9 April 2018|£337.50
23 April 2018|£675.00
```

Maybe we should fix the formatted approach here, to keep the PR focused, but maybe consider using the inline syntax so we don't litter PRs with whitespace changes?

```
| Leave                                                         | Key dates    |
| ------------------------------------------------------------- | ------------ |
| Start                                                         | 6 April 2018 |
| End                                                           | 4 April 2019 |
| Latest date to [claim leave](/employers-paternity-pay-leave/) |              |
```

See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables

Reply:
@andrewgarner regarding date formatting, the PR should be ok now. What about fixing tables in another PR? I am not sure if you are proposing that too. Thanks